.ci/release.yml

```
@@ -60,6 +71,13 @@ jobs:
       Write-Host "##$vstsCommandString"
     displayName: 'Capture PSResourceGet module NuGet package path and set environment variable'
 
+  - task: ManualValidation@0
+    displayName: Apporove release to PowerShell Gallery
```

nitpick/non-blocking
```suggestion
    displayName: Approve release to PowerShell Gallery
```